wpilibj/src/main/java/edu/wpi/first/wpilibj/AnalogPotentiometer.java

```diff
@@ -155,7 +155,7 @@ public void initSendable(SendableBuilder builder) {
   }
 
   /**
-   * Frees this resource.
+   * Release the resources used by this object.
    */
```

Comment:
The handling of comments for close() is inconsistent.  Some you’ve removed, others you’ve left or updated.

Reply:
I went through them again and removed the comments that said things like "Free the resources." that don't actually provide information in addition to what the reader already knows Java's Closeable interface is supposed to do.